tests/test_redshift/test_redshift.py

```diff
@@ -64,6 +65,31 @@ def test_create_cluster_with_enhanced_vpc_routing_enabled():
     response["Cluster"]["EnhancedVpcRouting"].should.equal(True)
 
 
+@mock_redshift
+def test_create_and_describe_cluster_with_kms_key_id():
```

Comment:
@bblommers I combined the create and describe into a single test to validate that the `KmsKeyId` is used. I added a check in the previous unit test to check that KmsKeyId is empty if not provided. If you'd rather me create another test with a signature of 

```python
test_create_and_describe_without_kms_key_id()
```

I can do that as well.